docs/sources/tutorials/send-metrics-to-prometheus.md

```diff
@@ -64,24 +65,23 @@ prometheus.relabel "filter_metrics" {
     source_labels = ["env"]
     regex         = "dev"
   }
-  
+
   forward_to = [prometheus.remote_write.metrics_service.receiver]
 }
 ```
 
-1. `prometheus.relabel` is a component most commonly used to filter Prometheus metrics or standardize the label set passed to one or more downstream receivers. 
```

Comment:
I would keep the original sentence here. 

IMO, filtering (dropping) Prometheus metrics and rewriting the label set of each metric mean two different things. 

The definition used in the original sentence is from the Prometheus.relabel page from the Grafana Alloy documentation:

https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.relabel/

<img width="614" alt="image" src="https://github.com/user-attachments/assets/c6d8b2b1-7a49-4507-88cc-da0dfb5c8106">